node/service/Cargo.toml

```diff
@@ -24,7 +24,7 @@ darwinia-staking-rpc-runtime-api    = { git = "https://github.com/darwinia-netwo
 darwinia-evm        = { git = "https://github.com/darwinia-network/darwinia-common.git", tag = "darwinia-v0.10.0" }
 dc-db               = { git = "https://github.com/darwinia-network/darwinia-common.git", tag = "darwinia-v0.10.0" }
 dc-mapping-sync     = { git = "https://github.com/darwinia-network/darwinia-common.git", tag = "darwinia-v0.10.0" }
-dc-rpc              = { git = "https://github.com/darwinia-network/darwinia-common.git", tag = "darwinia-v0.10.0" }
+dc-rpc              = { git = "https://github.com/darwinia-network/darwinia-common.git", tag = "darwinia-v0.10.0", features = ["rpc_binary_search_estimate"]}
```

Comment:
```
dc-rpc              = { features = ["rpc_binary_search_estimate"], git = "https://github.com/darwinia-network/darwinia-common.git", tag = "darwinia-v0.10.0" }
```

Reply:
Move the features section to the front? any special reason for this?

Reply:
Special items go first. Easy review.